<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1213</width>
    <height>628</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QTableView" name="tableView_2"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>学生</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>老师</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>工人</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QListView" name="listView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
